### ISSUE TITLE
Descriptions & Time slots

### ISSUE THREAD

**Issue by dbdimitrov** [saezlab/scheduling#6]
I really like this as an idea how to schedule our clubs but two major points seem to be pending (not sure if they were discussed in networks):

1. Concrete Definition of the clubs (e.g. Methods - how do we draw the line with Technical club).
2. Time slots are not obvious with this structure.

**Comment by slobentanzer** [comment]
Yeah, this is just a demo with dummy clubs. The real clubs need to be defined by the participants.
